src/abstractdataset/io.jl

```diff
@@ -44,18 +44,25 @@ function getmaxwidths(ds::AbstractDataset,
                       rowid::Union{Integer, Nothing},
                       show_eltype::Bool,
                       buffer::IOBuffer,
-                      truncstring::Int)
-    maxwidths = Vector{Int}(undef, size(ds, 2) + 1)
+                      truncstring::Int,
+                      mapformats::Bool)
```

Comment:
Here, make `mapformats` as a keyword argument with default value of `true`. Update the docstring of the function accordingly.